README.md

```diff
@@ -204,10 +204,10 @@ console.log(shoukaku.getIdealNode());
  Option | Type | Description
 --------|------|------------
 resume | boolean | Whether to resume a connection on disconnect to Lavalink (Server Side) (Note: DOES NOT RESUME WHEN THE LAVALINK SERVER DIES) |
-resumeKey | string | Resume key for Lavalink |
+sessionId | string | Session ID for lavalink to resume |
 resumeTimeout | number | Timeout before resuming a connection **in seconds** |
 resumeByLibrary | boolean | Whether to resume the players by doing it in the library side (Client Side) (Note: TRIES TO RESUME REGARDLESS OF WHAT HAPPENED ON A LAVALINK SERVER) |
-alwaysSendResumeKey | boolean | Disables the first time initialization tracking of nodes, and just sends the resume key always (Note: Useful for people who save their players to redis and wants to resume sessions even at first boot) |
+alwaysSendSessionId | boolean | Disables the first time initialization tracking of nodes, and just sends the session id always (Note: Useful for people who save their players to redis and wants to resume sessions even at first boot) |
```

Comment:
This needs to be reimplemented, since we expect shoukaku to be used from zero, hence if a user saved his sessionId. Can be deleted (in the whole code base) until we have a better implementation